### ISSUE TITLE
New Designs

### ISSUE THREAD

**Issue by maxgerber** [InFact-coop/LCN#48]
Hi Nimrod @NB76,

We have our ideas for the new designs/flow following on from our workshop and scoping conversation yesterday. Please take a look at it and record any comments here!

Thanks,
Max

**Comment by NB76** [comment]
Sure, but where am I looking? It's not the Elm Model, is it?

**Comment by maxgerber** [comment]
My bad! https://www.figma.com/file/eVz0BLuzez6XTDL9IdByRd4y/LCN this is it @NB76

**Comment by NB76** [comment]
Bad bad Max. I'll have a look at it with Julie soon and get back to you today.

**Comment by NB76** [comment]
Overall it looks very good already. Bearing out most of what we've talked about yesterday really feels like a big leap forward. Comments so far (only where we have them):

- Login: looks nice and clean, a good use of our brand identity. I take it the Law Centre name is just for the mock up? When logins are set up this would be preset. Or it could be an additional element to authenticate?
- Quant: are we off the slider idea or will this be implemented later on? Main thing is  to have the questions appear contextually, once you have made your selection of one of the three categories. I'd rename the first one 'Lawyer/Caseworker'
- Quant thanks: maybe rephrase second sentence as 'With your help, Law Centres have seen XX people this week'
- Comment: same question about the slider here too. I'll deal with copy separately but this is about right. Specify word limit
- Comment thanks: rename left button to 'add another comment' 
- Read comments + reply to comment: yup!
- See comment reply: good, as anticipated. We would need to think what would happen if people want to reply to replies. Not that they can, but because they cannot, would that not mess up the conversation? Just something to consider for now. 
- Logout: good! Maybe we could add a couple of links here to other Network resources (e.g. website), just to use engagement here to drum up engagement elsewhere, too. 

In all, it already looks good. Can't wait to see it develop.

**Comment by maxgerber** [comment]
Hi @NB76 !
Making good progress so far hooking things up to Airtable. In answer to your answer...

**Login**: Exactly. We are using a drop down menu for this. We'll pull the names of your Law Centres from your website unless you otherwise specify.
**Quant**: in regards to the sliders, Jen had a hunch that they could be potentially discouraging for people who haven't got through a lot of people that day. Currently we are using a number input with up and down buttons like this
![screen shot 2018-02-15 at 12 16 38 pm](https://user-images.githubusercontent.com/22657280/36256285-2b6f51b4-124a-11e8-92c5-34e9be0bb9d5.png). If you'd like to test the sliders with users, then let us know and we'll do those instead.
**Quant thanks**: :+1:
**Comment**: How do you imagine the sliders being implemented here? Do you mean the slider between the different Comment types?
**Comment thanks**: :+1: All copy changes highly encouraged
**Read comments + reply to comment**: Looks good, right?
**See comment reply**: Currently we're thinking that either people wouldn't be able to comment on comments with replies, or they could, but only the latest reply will be shown. All these comments will be sorted in order of time, and filtered by comment type. (comment)
**Logout**: Sure, we can look into that!

Glad you like it so far, it's really coming together quite nicely

**Comment by NB76** [comment]
Sounds good to me so far. Just a few comments:
**Login**: use the Law Centre names from the website for this stage. This might have to change for the final product, because the names on our website are populated from our SalesForce database. 
**Quant**: I'm agnostic about sliders, we can go with number input and change if testing raises issues.
**Comment**: yes, that's what I meant, but it could be another device, like tabs. Again, happy to go with your advice.
**Read comment + reply to comment**: I do like those!
**See comment reply**: Show latest reply only would be best. Could the filtering by comment type be made to look like tabbed browsing? So choosing 'success' would show category/heading and the comments below it as one tab? Right now the category buttons look a bit separate to the comments below them. Current colour as visual cue probably too subdued.
**Logout**: I'm just thinking of ways of making this stickier with other platforms people already use.

**Comment by ivanmauricio** [comment]
@NB76 Have you had any more thoughts about the links that you would like to go on the log out page?

**Comment by NB76** [comment]
Yes, @ivanmauricio! I was thinking of two main elements: 

- "Go to the [LCN website](www.lawcentres.org.uk)" 
- "Keep the conversation going" with links to our [Twitter account](https://twitter.com/LawCentres) and [Facebook page](https://www.facebook.com/LawCentres/) - the idea is that a caseworker, say, can share an issue she has seen internally, but also - in a different way - while it's fresh in her mind, be encouraged to talk about it publicly and raise awareness. Uptake will vary, but it's worth a try.
